Client/Frontend/Browser/Onboarding/OnboardingNavigationController.swift

```diff
@@ -47,12 +47,12 @@ class OnboardingNavigationController: UINavigationController {
         case shieldsInfo
         
         /// Returns new ViewController associated with the screen type
-        func viewController(with profile: Profile) -> OnboardingViewController {
+        func viewController(with profile: Profile, theme: Theme) -> OnboardingViewController {
```

Comment:
No, mostly just no reason for on-boarding to get a tab, just giving it the minimal of what it needs. At some point it may not even need that.

We may be refactoring on-boarding to be light/dark vs being created from a theme file.